spacelift/resource_scheduled_delete_stack.go

```diff
@@ -0,0 +1,171 @@
+package spacelift
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/shurcooL/graphql"
+
+	"github.com/spacelift-io/terraform-provider-spacelift/spacelift/internal"
+	"github.com/spacelift-io/terraform-provider-spacelift/spacelift/internal/structs"
+	"github.com/spacelift-io/terraform-provider-spacelift/spacelift/internal/validations"
+)
+
+func resourceScheduledDeleteStack() *schema.Resource {
+	return &schema.Resource{
+		Description: "" +
+			"`spacelift_scheduled_delete_stack` represents a scheduling configuration " +
+			"for a Stack. It will delete a stack at the given timestamp.",
+
+		CreateContext: resourceScheduledDeleteStackCreate,
+		ReadContext:   resourceScheduledDeleteStackRead,
+		UpdateContext: resourceScheduledDeleteStackUpdate,
+		DeleteContext: resourceScheduledDeleteStackDelete,
+
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"stack_id": {
```

Comment:
Is it worth including the `schedule_id` as well for consistency with the data source?

Reply:
addressed.